packages/cli/src/commands/alias/index.ts

```diff
@@ -12,7 +12,6 @@ import { AliasTelemetryClient } from '../../util/telemetry/commands/alias';
 import output from '../../output-manager';
 
 const COMMAND_CONFIG = {
-  default: ['set'],
```

Comment:
None of our other `COMMAND_CONFIG` have this structure which makes it fit poorly into the help pattern. We never match on `"default"` and if you check later in the method we branch of `default`.

Reply:
Extracted to https://github.com/vercel/vercel/pull/12438